scripts/Makefile

```diff
@@ -20,7 +20,12 @@ INTDIR:=../src
 OUTLIBDIR:=../src
```

Comment:
Looks like this will only modify the build if AFL_INSTRUMENT is present, correct?

Reply:
Exactly, I added this to prevent changing the behavior of a default build.